### PR TITLE
Update styles & recipes docs + rename template -> design settings

### PR DESCRIPTION
**Pull request by brittany-johnson**
In this PR I've updated a few of the documentation files, renamed the template directory to design-settings, and renamed template1, template2, and template3 to carnival, corn, and cardboard. After doing this I rebuild all of the styles and recipes.

### CODE REVIEW

**Comment by Stackblocks** [comment]
image

Starting in on this review but it'll take a bit 👍

**Review by ajluk — COMMENTED** [review]
@brittany-johnson I have noticed that after creating the directory for `desing-settings/<theme name>` you have also left the previous directory `templates/tamplate1-3/`. Was it intentional?

File: styles/output/anatomy-rex-web.css.map [hunk 1]
```diff
@@ -24,7 +24,7 @@
 		"../build/../framework/settings/_init.scss",
 		"../build/../framework/platform/_init.scss",
 		"../build/../framework/design-shape/_init.scss",
-		"../build/../templates/template1/_design.scss",
+		"../build/../design-settings/carnival/_design.scss",
```
Comment (wilkus27) [inline styles/output/anatomy-rex-web.css.map:27]: Do we still need those files with `rex-web`?

image

Only books from carnival theme have them. I don't remember what is the reason that they are not generated in corn and cardboard :thinking:

File: styles/templates/template1/_settings.scss [hunk 2]
```diff
@@ -3,24 +3,24 @@
 @import "typography";
 
 //template parts
-@import 'templates/template1/parts/link-settings';
-@import 'templates/template1/parts/_figureimage-settings.scss';
-@import 'templates/template1/parts/unit-settings.scss';
```
Comment (wilkus27) [inline styles/templates/template1/_settings.scss:8]: I think all files starts from this one to the end should be removed. They are in old directory

image

I'm not sure if this is what @ajluk meant in her comment, so I leave this note also here.

**Review by wilkus27 — CHANGES_REQUESTED** [review]
Wow :open_mouth: A lot of work is done here. I left only two comments. Beside that everything looks good :relaxed:

File: styles/templates/template1/_settings.scss (hunk 2 above)
Comment (ajluk) [reply styles/templates/template1/_settings.scss:8]: Yes, this is what I had in mind.  :+1:

**Review by Stackblocks — APPROVED** [review]

File: styles/output/anatomy-rex-web.css.map (hunk 1 above)
Comment (brittany-johnson) [reply styles/output/anatomy-rex-web.css.map:27]: Good question! At the beginning of the baked styles project, we intended on serving styles to REX, but that portion of the work was delayed. We'll be picking up on this again next year though.

File: books.txt [hunk 3]
```diff
@@ -78,55 +78,55 @@ BOOK_CONFIGS=(
 # "_example                   _example                           example-uuid                           col12345   easyvm5.cnx.org       mixins/                   example"
   "accounting-vol-1           accounting                         9ab4ba6d-1e48-486d-a2de-38ae1617ca84   col25448   easyvm5.cnx.org       mixins/"
   "accounting-vol-2           accounting                         920d1c8a-606c-4888-bfd4-d1ee27ce1795   col25479   easyvm5.cnx.org       mixins"
-  "additive-manufacturing     additive-manufacturing             d7f23ac9-cf1c-498a-ae49-f34de2cb379f   col10000   content03.cnx.org     mixins/                   template2"
-  "algebra-trigonometry       precalculus                        13ac107a-f15f-49d2-97e8-60ab2e3b519c   col11758   katalyst01.cnx.org    mixins/                   template2"
+  "algebra-trigonometry       precalculus                        13ac107a-f15f-49d2-97e8-60ab2e3b519c   col11758   katalyst01.cnx.org    mixins/                   corn"
```
Comment (Stackblocks) [inline books.txt:81]: Sorry to make you interact with this more, but would you please adjust these two so they maintain alphabetical order?

File: styles/README.md [hunk 4]
```diff
@@ -107,9 +107,9 @@ Component Variable Names are written using [Block, Element, Modifier (BEM) synta
 
 So given the example of `$note--boxed__container:` in the next section, the variable represents a container that is a piece of a "boxed" type note.
 
-### Writing the shapes
+### Writing the componenets and shapes
```
Comment (Stackblocks) [inline styles/README.md:110]: typo on "components" throughout

File: recipes/css-extensions.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
+### Easybake: Structured Text Manipulation via CSS Extensions
+(Taken from cnx-easybake repo)
+
+
```
Comment (Stackblocks) [inline recipes/css-extensions.md:4]: Please condense extra whitespace

**Review by Stackblocks — CHANGES_REQUESTED** [review]
A couple of editorial changes requested but otherwise looking good!

**Review by ajluk — APPROVED** [review]

**Review by kswanson33 — CHANGES_REQUESTED** [review]
Sneaking in a few technically-outside-the-PR change requests:

- styles/README.md line 158: current line reads 'map-merge($boxed_note_container, (`, change to `map-merge($Note--Boxed__Container, (`
- styles/README.md line 211: current line reads `import 'design/example/design';`, change to `import 'designs/example/design';`

**Pull request merged**